lib/src/configuration.cc

```diff
@@ -358,6 +359,17 @@ void Configuration::defineDefaultValues()
                     Types::Int,
                     5) } });
 
+    defaults_.insert(
+        Option { "allowed-keepalive-timeouts",
+                 Base_ptr { new Entry<int>(
+                    "allowed-keepalive-timeouts",
+                    "",
+                    lth_loc::translate("Number of consecutive keepalive response "
+                                       "timeouts to accept before assuming the "
```

Comment:
Oops, I changed my mind mid-way through that sentence. It's all going away for the `<hidden>` tag though.